src/everest/plugins/everest_plugin_manager.py

```diff
@@ -14,3 +16,7 @@ def __init__(self, plugins=None):
         else:
             for plugin in plugins:
                 self.register(plugin)
+
+    def get_documentation(self) -> Dict[str, Any]:
+        docs = self.hook.get_forward_model_documentations()
+        return docs[0] if docs else {}
```

Comment:
Why [0] here instead of merging dicts? Maybe non-relevant edge case but what if there were other installed python packages containing forward models for everest (currently only everest-models though)

Reply:
Will fix